### ISSUE TITLE
Null pointer exception on Project Creation with ADT 20

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#82]
```
Update to ADT 20 and create a project.

Null pointer exception is generated and PhoneGap project is not created.

Workaround: Use an older ADT.


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 2 Jul 2012 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
Workaround: Install older ADT

See 
http://stackoverflow.com/questions/4710107/where-can-i-find-older-versions-of-ec
lipse-adt for instructions for finding old ADT versions. I recommend 18.0.0
```

Original comment by `paul.beu...@gmail.com` on 2 Jul 2012 at 5:36

**Comment by GoogleCodeExporter** [comment]
```
planned for July 2012 1.2.91 release
```

Original comment by `paul.beu...@gmail.com` on 5 Jul 2012 at 2:37
- Added labels: **Milestone-Release1.2.91**

**Comment by GoogleCodeExporter** [comment]
```
Workaround is to put check box on Activity in the wizard.
```

Original comment by `paul.beu...@gmail.com` on 6 Jul 2012 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.2.91
```

Original comment by `paul.beu...@gmail.com` on 6 Jul 2012 at 5:45
- Changed state: **Fixed**
